domains/sree.json

```
@@ -0,0 +1,9 @@
+{
+  "owner": {
+    "username": "sree",
```

```suggestion
    "username": "sreekanthputta",
```
This must be your GitHub username.